src/ui/Panel/index.js

```diff
@@ -1,169 +1,160 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
```

Comment:
useEffect is not required here
```suggestion
import React, { useState } from 'react'
```